Assets/Mirror/Runtime/Transport/TelepathyTransport.cs

```diff
@@ -120,5 +117,18 @@ public int GetMaxPacketSize(int channelId)
             // Telepathy's limit is Array.Length, which is int
             return int.MaxValue;
         }
+
+        public override string ToString()
+        {
+            if (server.Active)
+            {
+                return "Telepathy Server port: " + port;
+            }
+            else if (client.Connecting || client.Connected)
+            {
+                return "Telepathy Client ip: " + client.client.Client.RemoteEndPoint + " port: " + port;
```

Comment:
remote endpoint already has the port.  it will show somehting like this:
If you have ipv6,  it iwll show like this:
```[::]:7777```
If you have ipv4,  it will show like this:
```127.0.0.1:7777```